json-rpc/src/stream_rpc/README.md

```diff
@@ -0,0 +1,79 @@
+# Streaming RPC
```

Comment:
might be useful to tell someone how this is enabled and point to the config and have the config point to this.